dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGeneratorTest.java

```diff
@@ -31,22 +28,37 @@
 @RunWith(Theories.class)
 public class StatsTaskListGeneratorTest {
 
+  static final StatsTaskListGenerator generator = new StatsTaskListGenerator();
+
   @DataPoints("nativeNames")
   public static final ImmutableList<String> nativeNames =
-      new StatsTaskListGenerator().nativeNames();
+      ImmutableList.<String>builder()
+          .addAll(generator.nativeNames(/* required= */ true))
+          .addAll(generator.nativeNames(/* required= */ false))
+          .build();
 
   @DataPoints("awrNames")
-  public static final ImmutableList<String> awrNames = new StatsTaskListGenerator().awrNames();
+  public static final ImmutableList<String> awrNames = generator.awrNames();
+
+  @DataPoints("statspackNames")
+  public static final ImmutableList<String> statspackNames = generator.statspackNames();
 
   @Theory
   public void nativeNames_allNamedFilesExist(@FromDataPoints("nativeNames") String name)
       throws IOException {
-    OracleStatsQuery.create(name, NATIVE, Duration.ofDays(7));
+    boolean isRequired = true;
+    OracleStatsQuery.createNative(name, isRequired, Duration.ofDays(7));
```

Comment:
Consider inlining the `isRequired` parameter and adding a parameter hint:
```
..., /* isRequired= */ true, ...
```

Similarly in other places.

Reply:
I like the idea here, but in some other places subjectively there's too much happening on a single line and spotless won't let me break it.

Done for GeneratorTest, similar places which combine this with a name string literal were left as is.